bmlab/controllers.py

```diff
@@ -531,6 +533,76 @@ def extract_payload_spectra(self, image_key):
         evm.set_spectra(image_key, spectra)
         return spectra, times, intensities
 
+    def get_maps(self, parameter_key):
+        """
+        This function returns the evaluated data,
+        its positions, dimensionality and labels
+        Parameters
+        ----------
+        parameter_key: str
+            The data type requested
```

Comment:
This is not a "data type", correct? It is a parameter key. How can I find out which parameters are available for an instance of EvaluationController?

Reply:
Yes, it's a data key. The available keys can be found via:
https://github.com/BrillouinMicroscopy/BMicro/blob/2a37f7907cbc57e4150e2c0d73aef6b084a1d29a/bmicro/gui/evaluation/evaluation_view.py#L184

Reply:
OK got it thanks. But please update this doc string here accordingly (maybe even referencing `get_parameter_list`.